h/streamer/websocket.py

```diff
@@ -71,7 +71,7 @@ def handle_message(message):
             jsonschema.validate(payload, filter.SCHEMA)
 
             # Add backend expands for clauses
-            _expand_clauses(payload)
+            _expand_clauses(socket.request, payload)
```

Comment:
If it's not, that's a very good point and we should probably pass `session` all the way through.